TRANSFORMERS/VECTOR_MANIPULATION/ROTATE_VECTOR/ROTATE_VECTOR.py

```diff
@@ -0,0 +1,26 @@
+from numpy import roll
+from flojoy import flojoy, Vector
+
+
+@flojoy
+def ROTATE_VECTOR(default: Vector, shift: int = 1) -> Vector:
```

Comment:
Is ROTATE the best name for this? Not sure what to call it but when I see rotate vector I think of actually rotating. Numpy roll:

https://numpy.org/doc/stable/reference/generated/numpy.roll.html

https://www.researchgate.net/publication/264422913/figure/fig1/AS:295910996234240@1447562087022/Rotation-of-a-vector-by-an-angle-in-2D-circular-coordinate-system.png

Reply:
Should I call it `SHIFT_VECTOR` then? `ROLL_VECTOR` doesn't seem that intuitive.

Reply:
Sure. I'm not happy with any of the names tbh haha

Reply:
Yeah let's come back to naming in a separate PR haha